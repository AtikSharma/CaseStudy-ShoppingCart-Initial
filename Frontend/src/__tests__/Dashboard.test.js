import React from "react";

import renderer from "react-test-renderer";

import Dashboard from './../NewUI/DashBoards/Dashboard';

afterAll(() => {
  jest.clearAllMocks();
});

test("renders correctly", () => {
  const tree = renderer.create(<Dashboard/>).toJSON();
  expect(tree).toMatchSnapshot();
});
