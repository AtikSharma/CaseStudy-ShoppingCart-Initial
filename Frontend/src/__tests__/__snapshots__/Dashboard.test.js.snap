// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="p-3 text-center"
>
  <div
    className="shadow m-auto rounded w-75  p-3 h-100 card"
    style={
      Object {
        "backgroundColor": "#F4FCD9",
      }
    }
  >
    <h5
      className="rounded shadow-sm text-center text-light card-header"
      style={
        Object {
          "backgroundColor": "#BB9981",
        }
      }
    >
      Dashboard
    </h5>
    <div
      className="card-body"
    >
      <ul
        class="list-group "
      >
        <li
          class="row"
        >
          <div
            class="col-3 rounded "
          >
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              UserId
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Name
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Email
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Gender
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Contact Number
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              DOB
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Role
            </li>
            <li
              class="list-group-item m-1 rounded shadow-sm fw-bold"
            >
              Address
            </li>
          </div>
          <div
            class="col-9 rounded"
          >
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <li
              class="list-group-item m-1 rounded shadow-sm"
            />
            <div
              class="list-group m-1 rounded shadow-sm "
            >
              <select
                class="form-select text-center"
              >
                 No Addresses
              </select>
            </div>
          </div>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
